### ISSUE TITLE
EMAIL QUEUE Mass update: missing the time update field

### ISSUE THREAD

**Issue by matteoraggi** [salesagility/SuiteCRM#3315]
Hi, today I have mass updated some emails i nthe queue, they was planned for today at time 16:00 and then I updated that queue for the same day, but the time changed by itself to 00:00, so then I discovered that it is missing the time field in the mass update and I suggest to add it to the next release, here the screenshot to explain it better:
![2017-03-24 19_16_12-eforte](https://cloud.githubusercontent.com/assets/14081756/24308126/20312550-10c7-11e7-88f4-8445ffa32bf0.png)

**Comment by samus-aran** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [EMAIL QUEUE Mass update: missing the time update field](https://trello.com/c/DMU7YG4c/11-email-queue-mass-update-missing-the-time-update-field)

**Comment by samus-aran** [comment]
We have now set up a new home for suggestions at Trello.  All github issues that were labeled 'suggestion' have been moved and will be closed.  Certain ones will be progressed within the new Suggestion Box and may be re-opened.

Announcement of moving Suggestions:
https://suitecrm.com/forum/suggestion-box/13691-moving-suggestions-from-github-to

New SuiteCRM Suggestion Box
https://trello.com/b/Ht7LbMqw/suitecrm-suggestion-box
